src/components/utils/blocks.ts

```diff
@@ -119,3 +138,93 @@ export function convertStringToBlockData(stringToImport: string, conversionConfi
     return {};
   }
 }
+
+/**
+ * Returns list of all available conversion menu items
+ *
+ * @param currentBlock - current block we need to get convetion data for
+ * @param api - editor API
+ */
+export async function getConvertToItems(currentBlock: Block, api: API): Promise<PopoverItemDefaultParams[]> {
```

Comment:
I'm not sure that it should be a util: 
- it depends on internal stuff (blockTools)
- it contains business logic (clear selection, setting a caret)

Reply:
moved this logic to conversion module